artifactory/services/upload.go

```diff
@@ -588,6 +593,8 @@ type UploadParams struct {
 	MinChecksumDeploy    int64
 	ChecksumsCalcEnabled bool
 	Archive              string
+	// When using 'archive' option we can set the target path in the archive using placeholders.
```

Comment:
I'm not sure the above description is clear enough. How about - 

// When using the 'archive' option for upload, we can control the target path inside the uploaded archive using placeholders. This operation determines the TargetPathInArchive value.